privacyidea/messages.pot

```diff
@@ -2427,107 +2438,112 @@ msgid ""
 "assigned."
 msgstr ""
 
-#: lib/tokens/pushtoken.py:71
+#: lib/tokens/pushtoken.py:72
 msgid "Please confirm the authentication on your mobile device!"
 msgstr ""
 
-#: lib/tokens/pushtoken.py:72
+#: lib/tokens/pushtoken.py:73
 msgid ""
 "Use the polling feature of your privacyIDEA Authenticator App to check "
 "for a new Login request."
 msgstr ""
 
-#: lib/tokens/pushtoken.py:74
+#: lib/tokens/pushtoken.py:75
 msgid "Do you want to confirm the login?"
 msgstr ""
 
-#: lib/tokens/pushtoken.py:367
+#: lib/tokens/pushtoken.py:368
 msgid "PUSH Token"
 msgstr ""
 
-#: lib/tokens/pushtoken.py:369
+#: lib/tokens/pushtoken.py:370
 msgid "PUSH: Send a push notification to a smartphone."
 msgstr ""
 
-#: lib/tokens/pushtoken.py:378
+#: lib/tokens/pushtoken.py:379
 msgid "The configuration of your Firebase application."
 msgstr ""
 
-#: lib/tokens/pushtoken.py:386
+#: lib/tokens/pushtoken.py:387
 msgid ""
 "The URL the Push App should contact in the second enrollment step. "
 "Usually it is the endpoint /ttype/push of the privacyIDEA server."
 msgstr ""
 
-#: lib/tokens/pushtoken.py:392
+#: lib/tokens/pushtoken.py:393
 msgid ""
 "The second enrollment step must be completed within this time (in "
 "minutes)."
 msgstr ""
 
-#: lib/tokens/pushtoken.py:396
+#: lib/tokens/pushtoken.py:397
 msgid "The smartphone needs to verify SSL during the enrollment. (default 1)"
 msgstr ""
 
-#: lib/tokens/pushtoken.py:402
+#: lib/tokens/pushtoken.py:403
 msgid "The user may only have this maximum number of Push tokens assigned."
 msgstr ""
 
-#: lib/tokens/pushtoken.py:407
+#: lib/tokens/pushtoken.py:408
 msgid "The user may only have this maximum number of active Push tokens assigned."
 msgstr ""
 
-#: lib/tokens/pushtoken.py:413
+#: lib/tokens/pushtoken.py:414
 msgid "Require to unlock the Smartphone before Push requests can be accepted"
 msgstr ""
 
-#: lib/tokens/pushtoken.py:419
+#: lib/tokens/pushtoken.py:420
 msgid ""
 "The question the user sees on his mobile phone. Several tags like "
 "{serial} and {client_ip} can be used as parameters."
 msgstr ""
 
-#: lib/tokens/pushtoken.py:426
+#: lib/tokens/pushtoken.py:427
 msgid "The title of the notification, the user sees on his mobile phone."
 msgstr ""
 
-#: lib/tokens/pushtoken.py:431
+#: lib/tokens/pushtoken.py:432
 msgid "The smartphone needs to verify SSL during authentication. (default 1)"
 msgstr ""
 
-#: lib/tokens/pushtoken.py:437
+#: lib/tokens/pushtoken.py:438
+msgid "Require the user to confirm the login with a presence check."
+msgstr ""
+
+#: lib/tokens/pushtoken.py:443
 msgid ""
 "The options that can be presented to the user to confirm the login. "
-"ALPHABETIC: A-Z, NUMERIC: 01-99, CUSTOM: user defined."
+"<code>ALPHABETIC</code>: A-Z, <code>NUMERIC</code>: 01-99, "
+"<code>CUSTOM</code>: user defined."
```

Comment:
I suppose there is no way we can get around these < code > tags in the translation, is there? 
There is a space missing after "user defined. ", because it is followed by another sentence which is not included in this translation.